components/uniset/src/props.rs

```diff
@@ -34,262 +34,327 @@ fn get_prop<'data, D: DataProvider<'data, UnicodePropertyV1Marker> + ?Sized>(ppu
 // Binary property getter fns
 //
 
+#[allow(missing_docs)] // TODO(#686) - Add missing docs.
```

Comment:
Suggestion: Create separate issues for each component.

Reply:
Done.